lib/grpc/server/interceptors/logger.ex

```diff
@@ -23,7 +23,7 @@ defmodule GRPC.Server.Interceptors.Logger do
         use GRPC.Endpoint
 
         # logs on :info and higher priority (warn, error...)
-        intercept GRPC.Server.Interceptors.Logger, level: :info, accepted_comparators: [:lt, :eq]
+        intercept GRPC.Server.Interceptors.Logger, level: :info, accepted_comparators: [:gt, :eq]
```

Comment:
In changing the default, we'll be able to remove the `accepted_comparators` altogether here